server/app.py

```diff
@@ -60,6 +60,34 @@ def createAccount():
 
     return jsonify(message="Successfully updated resource."), 200
 
+// Login
+@app.route("/api/Login", methods=["POST"])
+def Login():
+    username = request.get_json() ["username"]
+    password = request.get_json()["password"]
+
+    config = {
+        "user": "root",
+        "password": "sherlockeD123",
```

Comment:
Since this is a public repo I think we need to find a solution where we can maintain our codebase on Github without exposing any passwords. Would like to raise this to the team for our next standup.

Reply:
We actually have added these as env variables to the docker-compose.yml file so she can have non-hardcoded stuff here!

Reply:
Great! I see your fix below, Thanks!